src/pages/dbot/_dbot-video.js

```diff
@@ -64,98 +84,92 @@ const GoToLiveDemo = styled(Button)`
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            image_name="dbot-step2.png"
-                            current_step_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dbot-step2.png', 2)}
-                        >
-                            {localize('2. Set purchase conditions')}
-                        </Step>
-                    </Tab>
-                    <Tab>
-                        <Step
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            image_name="dbot-step3.png"
-                            current_step_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dbot-step3.png', 3)}
-                        >
-                            {localize('3. Set restart conditions')}
-                        </Step>
-                    </Tab>
-                    <Tab>
-                        <Step
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            image_name="dbot-step4.png"
-                            current_step_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dbot-step4.png', 4)}
-                        >
-                            {localize('4. Run bot')}
-                        </Step>
-                    </Tab>
-                    <Tab>
-                        <Step
-                            as="h4"
-                            lh="1.5"
-                            align="left"
-                            image_name="dbot-step5.png"
-                            current_step_image={this.state.current_step_image}
-                            onClick={() => this.clickHandler('dbot-step5.png', 5)}
-                        >
-                            {localize('5. Check profit')}
-                        </Step>
-                    </Tab>
-                    <GoToLiveDemo secondary="true" onClick={this.handleRedirect}>
-                        {localize('Go to live demo')}
-                    </GoToLiveDemo>
-                </TabsWrapper>
-                <VideoWrapper>
-                    <Image
-                        img_name={this.state.current_step_image}
-                        alt={localize('DBot - Step') + this.state.step}
-                        width="100%"
-                    />
-                </VideoWrapper>
-            </Container>
-        )
+const handleRedirect = () => {
+    window.open(deriv_bot_app_url, '_blank')
+}
+const DtraderTabs = () => {
+    const data = useStaticQuery(query)
+    const [current_step, setStep] = React.useState('step_1')
+    const clickHandler = (incoming_step) => {
+        if (incoming_step === current_step) return
```

Comment:
I think you don't need this equality check because React uses `Object.is` for checking the previous state with the current state before re-rendering the page. so passing repetitive values for primary data types (like strings) doesn't need an extra check.

Reply:
thanks, fiexed